File: problem_twenty_seven.py
# check sorted list
arr = int(input("Enter list length:"))
def MyFunction(arr):
    lst =[]
    for i in range(0,arr,1):
       element = int(input("Enter list element:"))
       lst.append(element)
       count =1
    for i in range(0,len(lst)-1,1):
        if i < i+1:
            count+=1
        else :
            print("No this element not sorted.")
            break   
    if len(lst)== count:
        print("Yes this list element sorted")

MyFunction(arr)

File: problem_twenty_seven.js
// checking sorted arr
arr =[1,2,3,4,5,9,7]
function myFunction(arr){
    var count = 1
    for(var i =0 ;arr.length -1>i;i++){
        if(arr[i]<arr[i+1]){
            count +=1
        }
        else{
            console.log("No this arry not sorted")
        }
    }
    if (count == arr.length){
        console.log("Yes this arry sorted")
    }
}
myFunction(arr)